templates/config/rbac/leader-election-role-binding.yaml.tpl

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: ack-system
```

Comment:
Use `namespace: {{ .Release.Namespace }}`. We don't know that the controller will be installed into `ack-system`.

Reply:
This is not a helm configuration, it's the plain manifests

Reply:
Ah I missed that. Nvm